scripts/delete-pod-disruption-budget.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source "$SCRIPT_DIR"/init-env.sh
+
+# Delete test PDB
+oc delete pdb test-pdb-min -n "${TNF_EXAMPLE_CNF_NAMESPACE}" --ignore-not-found=true
```

Comment:
Could we point these delete calls to the file `test-target/pod-disruption-budget.yaml`?  Instead of calling them out by name?

Reply:
Done.